hack/ansible-lint.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -x
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Author: Jorge A Gallegos <jgallego@redhat.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+#~Usage:
+#~  ansible-lint.sh [-f] [-p PROFILE]
```

Comment:
This seems outdated, getopts only picks cp:
```suggestion
#~  ansible-lint.sh [-c] [-p] 
```